Source/ACE/Entity/Player.cs

```diff
@@ -1558,7 +1573,42 @@ protected override void SendUpdatePosition()
             Session.Network.EnqueueSend(msg);
         }
 
-        public void HandleActionPutItemInContainer(ObjectGuid itemGuid, ObjectGuid containerGuid)
+        private void HandleUnequip(Container container, WorldObject item, uint location)
+        {
+            UpdateAppearance(container, item.Guid.Full);
+            Session.Network.EnqueueSend(new GameMessagePutObjectInContainer(Session, container.Guid, item, location),
+                new GameMessageUpdateInstanceId(item.Guid, container.Guid, PropertyInstanceId.Container),
+                new GameMessageObjDescEvent(this),
+                new GameMessageSound(Guid, Sound.UnwieldObject, (float)1.0));
+            CurrentLandblock.EnqueueBroadcast(Location, Landblock.MaxObjectRange, new GameMessageObjDescEvent(this));
+        }
+
+        private void HandleMove(WorldObject item, Container container, uint location)
+        {
```

Comment:
"HandleMove" in the Player class is a bit misleading going by the name alone.  Perhaps "MoveInventoryItem" or "HandleInventoryMove".